Sources/NIOPosix/BSDSocketAPICommon.swift

```diff
@@ -66,7 +66,7 @@ extension NIOBSDSocket.SocketType: Hashable {
 extension NIOBSDSocket.SocketType {
     /// Supports datagrams, which are connectionless, unreliable messages of a
     /// fixed (typically small) maximum length.
-    #if canImport(Glibc)
+    #if os(Linux) && !canImport(Musl)
```

Comment:
@MaxDesiatov, I presume this was your intent when you changed these two lines in this file.